src/defs.h

```diff
@@ -37,6 +37,7 @@
 #define MAX_CASES 128
 #define MAX_NESTING 128
 #define MAX_OPERAND_STACK_SIZE 32
+#define MAX_ANALYSIS_STACK_SIZE 700
```

Comment:
How can you measure and/or calculate the value?

Reply:
Loading an address from .data section increments the stack index of `live_kill` by 1 in analysis. The inline libc has ~700 times loading, so the limitation is set to 700.

Reply:
You should explain within git commit message(s).